easybuild/easyconfigs/m/MDSplus/MDSplus-7.7.12-intel-2018a.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'ConfigureMake'
+
+name = 'MDSplus'
+version = '7.7.12'
+
+homepage = 'http://mdsplus.org/'
+description = """MDSplus is a set of software tools for data acquisition and storage and a methodology
+ for management of complex scientific data."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
```

Comment:
@SimonPinches `opt` and `optarch` are enabled by default, so please trim to `toolchainopts = {'pic': True}`

Reply:
Okay, I'll fix and push a new version shortly...